provider/source/data/metaZonesPatched.json

```diff
@@ -127,6 +427,34 @@
                   }
                 }
               }
+            ],
+            "Istanbul": [
+              {
+                "usesMetazone": {
+                  "_mzone": "Europe_Eastern",
+                  "_to": "1978-06-28 21:00"
+                }
+              },
+              {
+                "usesMetazone": {
+                  "_mzone": "Turkey",
+                  "_from": "1978-06-28 21:00",
+                  "_to": "1984-10-31 23:00"
+                }
+              },
+              {
+                "usesMetazone": {
+                  "_mzone": "Europe_Eastern",
+                  "_from": "1984-10-31 23:00",
+                  "_to": "2016-09-06 21:00"
+                }
+              },
+              {
+                "usesMetazone": {
+                  "_mzone": "Turkey",
+                  "_from": "2016-09-06 21:00"
+                }
+              }
```

Comment:
Thought: We would save a lot of data if we mapped Turkey, Syria, Jordan, Saudi Arabia, and others all to the same metazone, maybe "Middle_East" or something. Maybe even throw in Moscow. :man_shrugging: